css/articles.css

```diff
@@ -0,0 +1,192 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: inherit; }
+
+  *::selection {
+    color: black;
+    background: #F4F5F5; }
+
+  *::after, *::before {
+    box-sizing: inherit; }
+
```

Comment:
kindly , indent well the codes like 
`*::after, *::before {
    box-sizing: inherit; 
}`